guides/common/modules/proc_using-freeipa-credentials-to-log-in-to-the-project-hammer-cli.adoc

```diff
@@ -1,60 +1,54 @@
 [id="Using_{FreeIPA-context}_credentials_to_log_in_to_the_{project-context}_Hammer_CLI_{context}"]
-= Using {FreeIPA} credentials to log in to the {Project} Hammer CLI
+= Logging in to Hammer CLI with {FreeIPA} credentials
```

Comment:
I need to look for the headings I've changed (there shouldn't be that many) and update the file names too.